lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,58 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+
+      let value = 'tab';
+      if (props.tab === props.selectedTab) {
+        value='tab active-tab';
+      } else {
+        value = 'tab'
+      }
+
+      console.log(props);
   return (
-    <div
-      className={''}
-      onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
-      }}
+    <StyledTab>
+      <button
+      className={value}
```

Comment:
a better way of of implementing this is the use of conditional rendering using the ternary operator:

`className={props.tab === props.selectedTab ? "tab active-tab" : "tab"}`